src/Service/TokenService.php

```
@@ -0,0 +1,229 @@
+    /**
+     * Validates an access token and returns the payload.
+     *
+     * @return array<string, mixed>
+     *
+     * @throws \InvalidArgumentException If the token is invalid
+     */
+    public function validateAccessToken(string $token): array
+    {
+        try {
+            $publicKey = $this->getPublicKey();
+            $decoded = JWT::decode($token, new Key($publicKey, self::ALGORITHM));
```

PHPMD (2.15.0)

Avoid using static access to class '\Firebase\JWT\JWT' in method 'validateAccessToken'. (undefined)

(StaticAccess)